### PR TITLE
menu added

### PR DESCRIPTION
**Pull request by balivarun**

### CODE REVIEW

**Comment by snapfast** [comment]
### <span aria-hidden="true">✅</span> Deploy Preview for *tiny-kangaroo-50ae23* ready!


|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 2d6dd1c19f9299cca28cd0c64c9e1192032b135f |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/tiny-kangaroo-50ae23/deploys/67277668ae7038000863b645 |
|<span aria-hidden="true">😎</span> Deploy Preview | [https://deploy-preview-3--tiny-kangaroo-50ae23.netlify.app](https://deploy-preview-3--tiny-kangaroo-50ae23.netlify.app) |
|<span aria-hidden="true">📱</span> Preview on mobile | <details><summary> Toggle QR Code... </summary><br /><br />QR Code<br /><br />_Use your smartphone camera to open QR code link._</details> |
---

_To edit notification comments on pull requests, go to your [Netlify site configuration](https://app.netlify.com/sites/tiny-kangaroo-50ae23/configuration/notifications#deploy-webhooks)._

**Comment by snapfast** [comment]
keep the color black and white website only.

File: src/app/component/header/header.component.css [hunk 1]
```diff
@@ -1,30 +1,46 @@
...
+    padding: 30px;
+    margin: 20px;
     display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-radius: 20px;
     flex-direction: row;
     align-items: center;
-    height: 50px;
+    background-color: rgb(236, 126, 57);
```
Comment (snapfast) [inline src/app/component/header/header.component.css]: Remove color -- keep it black and white.

File: src/app/component/header/header.component.css [hunk 2]
```diff
@@ -1,30 +1,46 @@
...
 
+.linguist-menu{
+    margin: 0;
+    padding-right: 20px;
+    margin-left: auto;
+}
 
-
+.menu-button{
+background-color: rgb(236, 126, 57);
```
Comment (snapfast) [inline src/app/component/header/header.component.css]: format this file

**Review by snapfast — APPROVED** [review]

File: src/app/component/header/header.component.css (hunk 1 above)
Comment (balivarun) [reply src/app/component/header/header.component.css]: done

**Review by snapfast — APPROVED** [review]
good

**Pull request merged**